### ISSUE TITLE
Laravel 5.3 not supported

### ISSUE THREAD

**Issue by rob1121** [barryvdh/laravel-async-queue#56]

**Comment by barryvdh** [comment]
Why not? What error do you get?

**Comment by rob1121** [comment]
Hi Sir Barry,
no email has been thrown.. Please have time to check my code below... 
Thanks in advance and regards,Rob


my notification class

my .env

queue.php


 

    On Tuesday, October 4, 2016 3:08 PM, Barry vd. Heuvel <notifications@github.com> wrote:
 

 Why not? What error do you get?—

**Comment by rob1121** [comment]
# Hi Sir Barry,

no email has been thrown.. Please have time to check my code below... 

### config/app.php
```
Barryvdh\Queue\AsyncServiceProvider::class
```

### config/queue.php
```
'default' => env('QUEUE_DRIVER', 'async'),

    'connections' => [

        'async' => [
            'driver' => 'async',
            'table' => 'jobs',
            'queue' => 'default',
            'binary'      => 'php',
            'binary_args' => '',
        ],
```
### .env
```
QUEUE_DRIVER=async
```

### web.php
```
Route::get("/", function() {
    $user = User::first();
    $spec = Companyspec::first();
    $user->notify(new InternalSpecUpdateNotifier($spec));
});
```

### InternalSpecUpdateNotifier.php

```
<?php

namespace App\Notifications;

use App\CompanySpec;
use Illuminate\Bus\Queueable;
use Illuminate\Notifications\Notification;
use Illuminate\Contracts\Queue\ShouldQueue;
use Illuminate\Notifications\Messages\MailMessage;

class InternalSpecUpdateNotifier extends Notification implements ShouldQueue
{
    use Queueable;

    private $spec;

    public function __construct(CompanySpec $spec)
    {
        $this->spec = $spec;
    }

    public function via($notifiable)
    {
        return ['mail'];
    }

    public function toMail($notifiable)
    {
        return (new MailMessage)
            ->subject("Internal spec update: {$this->spec->spec_no} - {$this->spec->name}")
            ->line("Revision summary:")
            ->line("{$this->spec->companySpecRevision->revision_summary}")
            ->action('View it now', url('/'))
            ->success()
            ->line('Feel free to contance me');
    }
}
```

**Comment by oriceon** [comment]
` [Symfony\Component\Debug\Exception\FatalThrowableError]
  Type error: Argument 3 passed to Illuminate\Queue\Worker::process() must be an instance of Illuminate\Queu
  e\WorkerOptions, none given, called in C:\SlotSys\www\aplicatie\vendor\barryvdh\laravel-async-queue\src\Co
  nsole\AsyncCommand.php on line 84`

**Comment by seiler-steinbach** [comment]
Any news on this one. I experience the same problem.

**Comment by Quezler** [comment]
same error as @oriceon here.

**Comment by Quezler** [comment]
And seeing that @rob1121 has in his web.php:

```
$user->notify(new InternalSpecUpdateNotifier($spec));
```

Is a fairly safe assumption that he uses [Spark's notification system](https://spark.laravel.com/docs/3.0/notifications)

**Comment by Quezler** [comment]
I have made a fix locally, and when i looked up the file in the repo, it appears @oriceon has fixed it in 21093d1.

The reason it does not fork for me, @seiler-steinbach and others,
is that the installation instructions say `composer require barryvdh/laravel-async-queue:"0.4.x"`
(which does not include the fix)

**so in short, here is the fix**:
`composer require barryvdh/laravel-async-queue dev-master`

(issue can be closed now @barryvdh)

**Comment by barryvdh** [comment]
Don't ever require dev-master please.
It has a branch alias, so you could use `0.5.x@dev`.

Also, I've tagged the current master as 0.5.0
